packages/multichain/src/caip25Permission.test.ts

```diff
@@ -493,7 +496,7 @@ describe('endowment:caip25', () => {
           },
         },
         normalizedOptionalScopes: {
-          'eip155:1': {
+          'eip155:5': {
```

Comment:
why?

Reply:
to match the input that validator() is called with (eip155:5 is passed in the optionalScopes, not eip155:1). This is of course unrelated to the validator changes being made in this PR